docs/standard/security/snippets/decrypting-data/csharp/aes-decrypt.cs

```diff
@@ -2,43 +2,40 @@
 using System.IO;
 using System.Security.Cryptography;
 
-public class DecryptExample
+try
 {
-    public static void Main(string[] args)
+    using FileStream fileStream = new("TestData.txt", FileMode.Open);
+
+    // Create a new instance of the default Aes implementation class
+    using Aes aes = Aes.Create();
+
+    // Reads IV value from beginning of the file.
+    byte[] iv = new byte[aes.IV.Length];
+    fileStream.Read(iv, 0, iv.Length);
```

Comment:
While I believe it always works for FileStream, Stream does not guarantee that the array gets filled in one call.  Not sure if it's worth complicating the sample for it, but on the other hand, it does lead to people writing bad calls to Stream.Read where they make bad assumptions.

Reply:
Good callout, I'm updating this now.